recipes/examl/build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+set -x
+set -e
+
+cd parser && make -f Makefile.SSE3.gcc && mv parse-examl ${PREFIX}/bin
+cd ../examl && make -f Makefile.SSE3.gcc && mv examl ${PREFIX}/bin
+rm -f *.o && make -f Makefile.AVX.gcc && mv examl-AVX ${PREFIX}/bin
+rm -f *.o && make -f Makefile.OMP.AVX.gcc && mv examl-OMP-AVX ${PREFIX}/bin
```

Comment:
@percyfal are those optimized versions for different architectures? Will they be compiled correctly on a machine that doesn't have the corresponding architecture? (I have no idea what travis-ci is running)